File: PERL/Cap06/c06ex04.pl
# c06ex04.pl

#!/usr/bin/perl

use strict;
use warnings;

print "Entre a quantidade de elementos da matriz: ";
my $N = <STDIN>;
chomp($N);
print "\n";

my @A;

for (my $I = 1; $I <= $N; $I++) {
  printf "Entre o %3do. nome: ", $I;
  chomp($A[$I] = <STDIN>);
}

print "\n";
print "Foram fornecidos os nomes:\n";
print "\n";

for (my $I = 1; $I <= $N; $I++) {
  printf "Nome %3d = %s\n", $I, $A[$I];
}

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: RUBY/Cap06/c06ex04.rb
# c06ex04.rb

#!/usr/bin/env ruby

aluno = []

print "Entre a quantidade de elementos da matriz: "
n = gets.chomp.to_i
puts "\n"

for i in 0..(n - 1)
  printf "Entre o %3do. nome: ", i + 1
  aluno << gets.chomp
end

puts "\nForam fornecidos os nomes:\n\n"

for i in 0..(n - 1)
  printf "Nome %3d = %s\n", i + 1, aluno[i]
end

print "\nTecle <Enter> para encerrar... "
gets
